From 52b4e74809eba796663f0f65194ab4f3eafe2cb1 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Fri, 3 Apr 2020 15:10:37 +0800
Subject: [PATCH] fix path to libffi

FFI_LIBRARY_PATH is the full path to libffi so when building something that links to
libLLVMInterpreter, we were looking for libffi in clang's recipe-sysroot instead of
the recipe's sysroot.

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 llvm/lib/ExecutionEngine/Interpreter/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/llvm/lib/ExecutionEngine/Interpreter/CMakeLists.txt b/llvm/lib/ExecutionEngine/Interpreter/CMakeLists.txt
index b8adea5..7d8434c 100644
--- a/llvm/lib/ExecutionEngine/Interpreter/CMakeLists.txt
+++ b/llvm/lib/ExecutionEngine/Interpreter/CMakeLists.txt
@@ -16,5 +16,5 @@ add_llvm_component_library(LLVMInterpreter
   )
 
 if( LLVM_ENABLE_FFI )
-  target_link_libraries( LLVMInterpreter PRIVATE ${FFI_LIBRARY_PATH} )
+  target_link_libraries( LLVMInterpreter PRIVATE ffi )
 endif()
-- 
2.7.4

